modules/integration/tests-ui-integration/src/test/resources/testng-server-mgt.xml

```diff
@@ -39,6 +39,7 @@
             <class name="org.wso2.carbon.mb.ui.test.log.ViewLogsTestCase"/>
             <class name="org.wso2.carbon.mb.ui.test.queues.BrowseQueueWithUserWithDomainName"/>
             <class name="org.wso2.carbon.mb.ui.test.configure.AdminWithDifferentAdminRoleName"/>
+            <class name="org.wso2.carbon.mb.ui.test.jira.MB939TestCase"/>
```

Comment:
Renamed to a more meaningful name with PR : https://github.com/wso2/product-mb/pull/41